rdma.go

```diff
@@ -34,13 +35,15 @@ func getAllRdmaDeivces() ([]Device, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	for _, d := range ibvDevList {
 		for _, n := range netDevList {
 			dResource, err := getRdmaDeviceResoure(d.Name)
 			if err != nil {
 				return nil, err
 			}
+			if strings.HasPrefix(n, "docker") || strings.HasPrefix(n, "cali") {
```

Comment:
I think put this to function `GetAllNetDevice` is better, WDYT?

Reply:
Done, thanks for review.